src/Tools/a2md/Program.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using System.Text;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace a2md
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            Loader loader = new Loader();
+            DescriptorEqualityComparer comparer = new DescriptorEqualityComparer();
+
+            var outputMarkdown = args
+                .Select(arg => new AnalyzerFileReference(arg, loader))
```

Comment:
It's possible, though more complicated that getting an analyzer because Code Fixes are MEF components. We would need to see how the Workspaces/Features layers of Roslyn handle that, and duplicate the approach.

We would also have to consider how to arrange them in relation to the diagnostics. A Code Fix may handle multiple different diagnostic IDs; different Code Fixes may handle the same ID; and a Code Fix may handle a diagnostic that is produced by a different package.

Still, probably worth doing in some form.